<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualCSV</class>
 <widget class="QMainWindow" name="ManualCSV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>433</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: #171717;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="twod_plot">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #171717;
color: white;
border-radius: 5px;
}
QPushButton:hover {
    background-color: #fd8a44; 
    border-color: #fd8a44; 
}</string>
    </property>
    <property name="text">
     <string>Plot 2D Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #171717;
color: white;
border-radius: 5px;
}
QPushButton:hover {
    background-color: #fd8a44; 
    border-color: #fd8a44; 
}</string>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>941</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #212121;
color: white;
border-radius: 5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>15</number>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Days</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Reciever 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Reciever 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Reciever 3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>15</string>
     </property>
    </column>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>2</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>3</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>4</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>5</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>6</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string>7</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="7">
     <property name="text">
      <string>8</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="8">
     <property name="text">
      <string>9</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="9">
     <property name="text">
      <string>10</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="10">
     <property name="text">
      <string>11</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="11">
     <property name="text">
      <string>12</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="12">
     <property name="text">
      <string>13</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="13">
     <property name="text">
      <string>14</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="14">
     <property name="text">
      <string>15</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="7">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="7">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="5">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="7">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #212121;
color: white;
border-radius: 5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="output_screen">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #171717;
color: white;
border-radius: 5px;
</string>
    </property>
    <property name="text">
     <string>Enter your data then save it</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="threed_plot">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>340</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #171717;
color: white;
border-radius: 5px;
}
QPushButton:hover {
    background-color: #fd8a44; 
    border-color: #fd8a44; 
}</string>
    </property>
    <property name="text">
     <string>Plot 3D Data</string>
    </property>
   </widget>
   <zorder>label_1</zorder>
   <zorder>save_button</zorder>
   <zorder>twod_plot</zorder>
   <zorder>table</zorder>
   <zorder>label</zorder>
   <zorder>output_screen</zorder>
   <zorder>threed_plot</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
